### PR TITLE
add data source user

### PR DESCRIPTION
**Pull request by psg2**
Issue: https://github.com/databrickslabs/terraform-provider-databricks/issues/647

### CODE REVIEW

File: identity/data_user.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+
+// DataSourceUser returns information about user specified by user name
+func DataSourceUser() *schema.Resource {
+	type entity struct {
+		UserName             	string   `json:"user_name"`
+		Home             	string   `json:"home" tf:"computed"`
```
Comment (nfx) [inline identity/data_user.go:19]: Please run make fmt

File: identity/data_user.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		Alphanumeric             	string   `json:"alphanumeric" tf:"computed"`
+	}
+
+	s := common.StructToSchema(entity{}, func(
```
Comment (nfx) [inline identity/data_user.go:23]: By the way, if you feel that schema map is shorter - you can do it as well :) just a side thought

File: identity/data_user.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		s map[string]*schema.Schema) map[string]*schema.Schema {
+		// nolint once SDKv2 has Diagnostics-returning validators, change
+		s["user_name"].ValidateFunc = validation.StringIsNotEmpty
```
Comment (nfx) [inline identity/data_user.go:26]: We've migrated to other linter, so comment above might not work. You need ValidateDiagFunc and wrap this sdkv1 one

**Review by nfx — CHANGES_REQUESTED** [review]
Small fixes and we're ready

File: identity/data_user.go (hunk 1 above)
Comment (psg2) [reply identity/data_user.go]: Done.

File: identity/data_user.go (hunk 2 above)
Comment (psg2) [reply identity/data_user.go]: Done.

File: identity/data_user.go (hunk 3 above)
Comment (psg2) [reply identity/data_user.go]: I guess it's done now that I've changed to schema map.

**Review by nfx — APPROVED** [review]
Thank you

**Pull request merged**